index.html

```diff
@@ -9,25 +9,62 @@
     <title>AwesomeBooks</title>
   </head>
 <body>
-  <h1 class="d-flex justify-content-center">All Awesome books</h1>
-    <div id="bookshelf"></div>
-    <br />
-    <form id="addbook" class="d-flex justify-content-center">
-      <h2>Add a new book</h2>
-      <div>
-        <label for="title"></label>
-        <input type="text" id="title" placeholder="Title" />
-      </div>
-      <br />
-      <div>
-        <label for="author"></label>
-        <input type="text" id="author" placeholder="Author" />
-      </div>
-      <br />
-      <button id="btn" >Add</button>
-    </form>
-    <script src="./scripts.js"></script>
-  </body>
+<header>
+  <nav class="navigation d-flex justify-content-between align-items-center">
+    <h2 class="logo ms-3">Awesome Books</h2>
+    <ul class="d-flex gap-3">
+      <li><a id="booksBtn" href="#">List</a></li>
+      <li><a id="addbookBtn" href="#">Add NewBook</a></li>
+      <li><a id="contactBtn" href="#">Contact</a></li>
+    </ul>
+  </nav>
+</header>
+
+<main>
```

Comment:
- [ ] When any of the links are clicked, the main area changes to display only one of the three content sections but for now your main area is showing all three sections, you can add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods

![image](https://user-images.githubusercontent.com/75126481/165754621-e4e0e8c9-da82-476f-ad2c-f82df6981961.png)